src/core/dev/cq_mgr_rx_regrq.cpp

```diff
@@ -117,7 +117,9 @@ void cq_mgr_rx_regrq::cqe_to_mem_buff_desc(struct xlio_mlx5_cqe *cqe,
         p_rx_wc_buf_desc->rx.tls_decrypted = (cqe->pkt_info >> 3) & 0x3;
 #endif /* DEFINED_UTLS */
         p_rx_wc_buf_desc->rx.timestamps.hw_raw = ntohll(cqe->timestamp);
-        p_rx_wc_buf_desc->rx.flow_tag_id = ntohl((uint32_t)(cqe->sop_drop_qpn));
+        uint32_t sop_rxdrop_qpn_flowtag_h_byte = ntohl((uint32_t)(cqe->sop_rxdrop_qpn_flowtag));
```

Comment:
I think the explicit cast to `uint32_t` is not required. Same for the strq file.